src/main/java/com/cms/claimmanagement/surveyor/SurveyorService.java

```diff
@@ -0,0 +1,21 @@
+package com.cms.claimmanagement.surveyor;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Optional;
+
+public class SurveyorService {
+    @Autowired
+    public SurveyorRepository surveyorRepository;
+    public String getSurveyorById(String claimId) {
+
+         Optional surveyorOpt = surveyorRepository.findById(claimId);
+         boolean surveyorExists = surveyorOpt.isPresent();
+         if (surveyorExists){
+             return "surveyor does not exists";
```

Comment:
I know it's not implemented yet, but _if(surveyorExists)_ I would expect the "There's a surveyor" message.